lib/v2/screens/sign_up/claim_invite/components/invite_link_fail_dialog.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/i18n/security.i18n.dart';
+import 'package:seeds/v2/components/custom_dialog.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+
+class InviteLinkFailDialog extends StatelessWidget {
+  const InviteLinkFailDialog({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return CustomDialog(
+      icon: const Icon(Icons.fingerprint, size: 52, color: AppColors.green1),
```

Comment:
@gguij004 could you put the correct figma icon? I don't have the time for that anymore.